### DISCUSSION TITLE
Team QB Slot

### DISCUSSION THREAD

**Discussion by MrDGer** [cwendt94/espn-api#477] [ESPN API]
We are trying out the Team QB position this year rather than the regular QB.  I was going through the discussions and didn't see something that would allow that to be brought up.

Last year, I was using this to call it

box_scores[posmatchuprosternum].home_lineup[posrosterspots].slot_position == 'QB'

Is there another call besides QB that would bring up the team QB?

**Comment by cwendt94** [comment]
I haven't heard of Team QB position before, is this a new scoring setting or League type? I would have to learn about it a little to figure out how to show the correct data for your league.
